docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotEncoding.cs

```diff
@@ -73,13 +70,43 @@ public static void Example()
             // 2
             // 3
 
+            // Multi column example : A pipeline for one hot encoding two columns 'Education' and 'ZipCode' 
+            var multiColumnKeyPipeline = mlContext.Transforms.Categorical.OneHotEncoding(
+                new InputOutputColumnPair[] {
+                    new InputOutputColumnPair("Education"),
+                    new InputOutputColumnPair("ZipCode"),
+                });
+
+            // Fit and Transform data.
+            var transformedData = multiColumnKeyPipeline.Fit(data).Transform(data);
+
+            var convertedData = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, true);
+
+            Console.WriteLine("One Hot Encoding of two columns 'Education' and 'ZipCode'.");
+            foreach (var item in convertedData)
+                Console.WriteLine("{0}\t\t\t{1}", string.Join(" ", item.Education), string.Join(" ", item.ZipCode));
+
+            // 1 0 0                   1 0
+            // 1 0 0                   0 1
+            // 0 1 0                   1 0
+            // 0 1 0                   0 1
+            // 0 0 1                   1 0
         }
```

Comment:
make it a separate example, because the multi-output is a different API.

Reply:
made separate example for multi input

---
In reply to: [271811670](https://github.com/dotnet/machinelearning/pull/3179#discussion_r271811670) [](ancestors = 271811670)